packages/client/src/constants/maps.js

```diff
@@ -49,3 +70,17 @@ export const SAFE_TYPES_TO_META = {
     displayText: "Other",
   },
 };
+
+/** 
+ * TODO: Remove once API for fetching the name of the token is implemented
+ * For now every time the fungible token is added to flow.json, the token name should be added here 
+ * (in case it is different from the contract name) 
+*/
+export const CONTRACT_NAME_TO_COIN_TYPE = {
+  "FlowToken": "FLOW",
```

Comment:
Let's use the enums defined above
```suggestion
  "FlowToken": COIN_TYPES.FLOW,
  "FUSD":COIN_TYPES.FUSD
  ...
```

Reply:
Updated!

Reply:
I don't see the updates, did you push up your change?

Reply:
https://github.com/DapperCollectives/VESSEL/pull/64/files#diff-784256b57bc20779b89d00fe1a77c943db2bd4e174e263610990518f2293b560R79-R85:~:text=%22FiatToken%22%3A-,COIN_TYPES,-.USDC%2C